fbw-a32nx/src/systems/instruments/src/PFD/LandingSystemIndicator.tsx

```diff
@@ -741,3 +747,59 @@ class MarkerBeaconIndicator extends DisplayComponent<{ bus: ArincEventBus }> {
     );
   }
 }
+
+class LsReminderIndicator extends DisplayComponent<{ bus: ArincEventBus }> {
+  private readonly sub = this.props.bus.getSubscriber<PFDSimvars & FcuBus & FgBus>();
+
+  private readonly glsMlsFlsOrLocVnavInstalled = Subject.create(false);
+
+  private readonly fwcFlightPhase = ConsumerSubject.create(this.sub.on('fwcFlightPhase'), 0);
+
+  private readonly fmgcFlightPhase = ConsumerSubject.create(this.sub.on('fmgcFlightPhase'), FmgcFlightPhase.Preflight);
+
+  private readonly fmgcDiscreteWord2 = Arinc429ConsumerSubject.create(this.sub.on('fmgcDiscreteWord3'));
+
+  private readonly fmgcDiscreteWord4 = Arinc429ConsumerSubject.create(this.sub.on('fmgcDiscreteWord4'));
+
+  private readonly landModeArmedOrActive = MappedSubject.create(
+    ([fmgcDiscreteWord2, fmgcDiscreteWord4]) =>
+      fmgcDiscreteWord4.bitValueOr(14, false) || fmgcDiscreteWord2.bitValueOr(20, false),
+    this.fmgcDiscreteWord2,
+    this.fmgcDiscreteWord4,
+  );
+
+  private readonly fcuEisDiscreteWord2 = Arinc429ConsumerSubject.create(this.sub.on('fcuEisDiscreteWord2'));
+
+  private readonly lsPushed = this.fcuEisDiscreteWord2.map((w) => w.bitValueOr(22, false));
+
+  private readonly lsReminderVisible = MappedSubject.create(
+    ([fwcPhase, fmgcPhase, landModeArmedOrActive, lsPushed]) => {
+      return (
+        landModeArmedOrActive &&
+        fmgcPhase === FmgcFlightPhase.Approach &&
+        !lsPushed && // TODO Check if LOC or G/S scales are invalid
+        fwcPhase !== 8 &&
```

Comment:
Why are we checking for FWC flight phases here? I don't see why that is necessary

Reply:
Disregard, sorry, just checked the ref again for the conditions